src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,31 @@
-export const TodoInfo = () => {};
+import { FC } from 'react';
+import { Todo } from '../../api/todos';
+import { User } from '../../api/users';
+import { UserInfo } from '../UserInfo';
+import usersFromServer from '../../api/users';
```

Comment:
The `usersFromServer` data should not be imported directly here. According to the task requirements, data should be prepared in the `App` component and passed down to child components. Consider passing the necessary user data as a prop to `TodoInfo`.